[2025-08-03T10:58:02.090+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:58:02.091+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:58:02.134+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:58:02.134+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:58:02.186+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:57:00+00:00 [queued]>
[2025-08-03T10:58:02.197+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:57:00+00:00 [queued]>
[2025-08-03T10:58:02.197+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-08-03T10:58:02.213+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_training_data> on 2025-08-03 10:57:00+00:00
[2025-08-03T10:58:02.223+0000] {standard_task_runner.py:57} INFO - Started process 1174 to run task
[2025-08-03T10:58:02.227+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'fetch_training_data', 'scheduled__2025-08-03T10:57:00+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpcenzzvzg']
[2025-08-03T10:58:02.231+0000] {standard_task_runner.py:85} INFO - Job 106: Subtask fetch_training_data
[2025-08-03T10:58:02.303+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:58:02.303+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:57:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:58:02.306+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:57:00+00:00 [running]> on host 407839faddcf
[2025-08-03T10:58:02.419+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='fetch_training_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-03T10:57:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-03T10:57:00+00:00'
[2025-08-03T10:58:02.431+0000] {db_utils.py:17} INFO - Connected to PostgreSQL
[2025-08-03T10:58:03.777+0000] {train_predict_to_db_dag.py:29} INFO - Fetched training data tail:
            id  store  ...  productname                 created_at
100678  156118     37  ...    product_A 2025-08-03 10:57:14.593698
100679  156119     38  ...    product_A 2025-08-03 10:57:24.593512
100680  156120     39  ...    product_A 2025-08-03 10:57:34.609180
100681  156121     40  ...    product_A 2025-08-03 10:57:43.989627
100682  156122     41  ...    product_A 2025-08-03 10:57:53.996415

[5 rows x 12 columns]
[2025-08-03T10:58:03.824+0000] {python.py:194} INFO - Done. Returned value was: {'store': 370, 'start_date': '2025-05-05', 'end_date': '2025-08-03'}
[2025-08-03T10:58:03.862+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=fetch_training_data, execution_date=20250803T105700, start_date=20250803T105802, end_date=20250803T105803
[2025-08-03T10:58:03.949+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-03T10:58:03.983+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
